doc/ChangeLog

```diff
@@ -1,4 +1,231 @@
 ===============================================================
+Tag name: ctsm5.2.0
+Originator(s): erik (Erik Kluzek)
+Date: Wed Apr  3 01:17:18 MDT 2024
+One-line Summary: New surface datasets and new mksurfdata_esmf tool to create them
+
+Purpose and description of changes
+----------------------------------
+
+Changes to CTSM:
+================
+
+All new surface datasets. Transient urban and lake by default turned on for transient cases.
+Ocean is run as baresoil rather than wetland (for clm6_0). The urban streams file was also
+updated. The new surface datasets have new updated input datasets (see below).
+
+New surface datasets:
+=====================
+
+The new surface datasets are incompatible with previous versions (for example the ctsm5.1 series)
+ctsm5.2.0 and following versions also can NOT use the previous ctsm5.1 datasets.
+
+See the section below about the new datasets used in their creation. Improvements in how landunits
+on coastal areas was also done.
+
+The following fields were added and removed to the list of fields on the datasets.
+
+Fields added:
+   ORGC, BULK, CFRAG, PHAQ (soil data)
+   mapunits (map units for the soil dataset)
+   LANDFRAC_MKSURFDATA
+   PCT_OCEAN (previously PCT_WETLAND was used)
+
+Fields removed:
+   AREA, PFTDATA_MASK
+
+New mksurfdata_esmf Tool:
+=========================
+
+Implement a parallel version of mksurfdata (mksurfdata_esmf) that uses ESMF regridding directly
+in mksurfdata so that offline mapping files don't have to be created as a separate step. This
+allows mksurfdata to create surface datasets at much higher resolutions
+
+The build for the tool is based on the CESM/CIME build system and uses cmake. This allows the build
+to be kept up with changes in CESM. Currently it's only setup and working on Derecho. But, this design
+will enable it to be built and run on any CESM supported machine (or a machine that a user ports to).
+
+Any input grid from ccs_config can be used, or the user can supply their own mesh file to define
+the output grid. The user no longer has to add to the list of valid resolutions as in mksurfdata_map.
+
+Creation of supported single point datasets. These datasets are created through the use of subset_data.
```

Comment:
I would add here:
For regional grids we also recommend subset_data due to issue #2430

Reply:
@slevis-lmwg I've added changes per your suggestions here:

https://github.com/ESCOMP/CTSM/pull/2464/files#diff-dffb4c6ae3432fc634c2a4cf112c586eccc9ebd5a5f428ebeeb106452821bc5d